README.md

```diff
@@ -1,10 +1,260 @@
-# adminrouter
-The admin router is an open-source Nginx configuration created by Mesosphere that provides central authentication and proxy to DCOS services within the cluster.
+# Adminrouter
 
+The Adminrouter is an open-source Nginx configuration created by
```

Comment:
Thanks :). The primary goal is to use the same terminology across _documentation_. This is an ongoing effort since about a year. Also see https://github.com/dcos/dcos-docs/pull/201.